content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -5,7 +5,7 @@ reviewers:
 - liggitt
 title: Using RBAC Authorization
 content_template: templates/concept
-aliases: [/rbac/]
+aliases: ["/rbac/","/docs/rbac"]
```

Comment:
Hi @jimangel . 
Just curious, do other localized versions of this page add the same `aliases` attribute?

Reply:
Looking here: https://bwaycer.github.io/hugo_tutorial.hugo/extras/aliases/

They give an example of explicit localization aliases if needed. (ex: `/zh/docs/rbac` and `/zh/rbac`). They also make it clear that no "assumptions" are made.

Given that the intent of these short urls is for sharing security information easily, and there are no referential links, the decision could be left up to the localization team if they wish to implement.

@tengqm @zacharysarah thoughts?

Reply:
Just looked further, and it appears that there is a way to make it [page-relative](https://gohugo.io/content-management/urls/#example-aliases-in-multilingual)

>From Hugo 0.55 you can also have page-relative aliases, so /es/posts/my-original-post/ can be simplified to the more portable my-original-post/

I will play with this a little locally.

Reply:
More info: https://github.com/gohugoio/hugo/issues/5757

Reply:
@jimangel Do you want to keep working on this, or merge and iterate?

Reply:
so ... reading the hugo docs, maybe you want to use the relative path, @jimangel ?

Reply:
Correct, I've updated the PR to use the relative path. This will make it so localization teams do not need to update the alias during their translation efforts. I did remove the `/docs/` link as I believe the benefit is not worth the effort to maintain.

This allows the scope to remain small: root level, security related content (and hopefully prevents this from becoming an undefined standard).